bin/build_structure_file.rb

```diff
@@ -108,7 +108,30 @@ def merge_structures(structures, additions)
   iban_registry_file = CSV.read(
     File.expand_path('../../data/raw/IBAN_Registry.txt', __FILE__),
     col_sep: "\t",
-    headers: true
+    headers: false,
+    encoding: "ISO8859-1",
+  )
+
+  # this thing is rotated from what normal people would do
+  transformed_registry_file = []
+
+  iban_registry_file.each_index do |row|
```

Comment:
Can we use [transpose](https://docs.ruby-lang.org/en/2.3.0/Array.html#method-i-transpose) here?